Client/src/app/login/login.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.css']
+})
+export class LoginComponent {
+  modal ={ 
```

Comment:
Add proper alignment for the code. A whitespace should be added before and after using the `=` operator.